Gemfile

```diff
@@ -9,7 +9,7 @@ gem "hashie", "~> 3.4"
 gem "govspeak", "~> 3.4"
 
 gem "gds-api-adapters", "23.2.2"
-gem "govuk_template", "~> 0.16"
+gem "govuk_template", "0.16.2"
```

Comment:
I'd vote for keeping the pessimistic versioning here and just running `bundle update govuk_template` instead.